rustls-mbedpki-provider/src/client_cert_verifier.rs

```diff
@@ -62,6 +66,43 @@ impl MbedTlsClientCertVerifier {
     pub fn root_subjects(&self) -> &[DistinguishedName] {
         self.root_subjects.as_ref()
     }
+
+    /// Sets the optional verification callback function for the certificate verification process.
+    ///
+    /// The verification callback allows you to customize how the certificate verification is performed.
+    ///
+    /// # Arguments
+    ///
+    /// * `callback` - A trait object implementing the `mbedtls::x509::VerifyCallback` trait, wrapped in an `Arc`.
+    pub fn set_verify_callback(&mut self, callback: Arc<dyn mbedtls::x509::VerifyCallback + 'static>) {
```

Comment:
Can this take an `Option`? I'm thinking of trying to reset `verify_callback`

Reply:
sure

Reply:
done!